scenarioo-client/app/step/screenAnnotations/screenAnnotationsButton.html

```diff
@@ -1,7 +1,7 @@
 <!-- View Screen Annotations toggle button -->
-<div style="display:inline-block;" id="sc-showHideScreenAnnotationsButton" ng-click="screenAnnotationsButton.toggleAnnotationsVisible()" ng-show="screenAnnotations">
+<div style="display:inline-block;" id="sc-showHideScreenAnnotationsButton" ng-click="screenAnnotationsButton.toggleAnnotationsVisible()" ng-show="screenAnnotations.length">
```

Comment:
I think it would be better to implement a 'hasScreenAnnotations' function on the directive that encapsulates an expression that also checks screenAnnotations to not being null or undefined - that would be more robust - otherwise this code relies on the list always to be an array (did you check that this is really the case?)